Cargo.toml

```diff
@@ -0,0 +1,59 @@
+# Templated by `cargo-generate` using https://github.com/danielhenrymantilla/proc-macro-template
+[package]
+name = "maybe-dangling"
+authors = [
+    "Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>",
+]
+version = "0.1.0"  # Keep in sync
+edition = "2021"
+rust-version = "1.65.0"
+
+license = "Zlib OR MIT OR Apache-2.0"
+repository = "https://github.com/danielhenrymantilla/maybe-dangling.rs"
+documentation = "https://docs.rs/maybe-dangling"
+readme = "README.md"
+
+description = "`ManuallyDrop<T>` and `MaybeDangling<T>` semantics in stable Rust as per https://github.com/rust-lang/rfcs/pull/3336"
+keywords = ["unsafe", "dangling", "aliasing", "dereferenceable", "manuallydrop"]
+
+[features]
+default = [
+]
+
+# Enable and use `#![feature(dropck_eyepatch)]` for `MaybeDangling`.
+nightly-dropck_eyepatch = []
+
+# unstable! Will probably use nightly to improve doc quality
+better-docs = []
+
+ui-tests = [
+    "better-docs",
+]
+
+# `docs-rs` is very similar to `better-docs`, but for it being allowed to enable
+# other features as well, often to make sure optional/non-`default` features
+# make it to the rendered docs (using `--all-features` instead is not that great).
+docs-rs = [
+    "better-docs",
+]
```

Comment:
Does it not make more sense for this to be a `--cfg` flag?

Reply:
I've seen this pattern quite a bit, true, but I somehow find the feature to be more convenient, provided that it is properly document as unstable. But I have no strong opinions on this either way 🙂

Reply:
I'm used to `cargo build --all-features` working on Tokio with a stable compiler, which is why we use a `--cfg`. But I guess you also have other unstable features here.

Reply:
Yeah, currently the `default-features` play the role of the `--all-features` of finer-featured crates such as `tokio`; I wouldn't expect us going that far with this crates, but if we do, I suspect we'd end up with a `all-stable-features` Cargo feature for this (which is partially orthogonal to the specific case of docs.rs tweaks, since these are not expected to be used by users, so a `--cfg` can make a lot of sense there).

If you feel like making the change for these docs.rs knobs anyways feel free to do so or suggested it over https://github.com/danielhenrymantilla/maybe-dangling.rs/pull/2

Reply:
It doesn't seem like `better-docs` or `docs-rs` are used at all? Maybe I'm missing something.

Reply:
Ah true 😄 This is from a template I use, which lets me opt into extra facilities. For instance, it has recently occurred to me that we could consider feature-gating the `MaybeDangling` wrapper (since I expect to be rarely needed to have something with inherent drop glue and yet allowed to dangle), and in that case the idea would be to use `better-docs` to enable `doc(cfg())`